leaders/src/main/java/edu/cornell/resultsmanager/TestOutputSender.java

```diff
@@ -0,0 +1,45 @@
+package edu.cornell.resultsmanager;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@Slf4j
+public class TestOutputSender {
+
+    public static void sendResults(String json, String url) {
+        try {
+            URL postUrl = new URL(url);
+            HttpURLConnection connection = (HttpURLConnection) postUrl.openConnection();
+            connection.setRequestMethod("POST");
+            connection.setDoOutput(true);
+            connection.setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
+
+            String postData = "results=" + java.net.URLEncoder.encode(json, "UTF-8");
```

Comment:
Just to keep within the style we've been using, I would recommend importing `URLEncoder` instead of using its full path